omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp

```diff
@@ -110,15 +115,18 @@
     </div>
 </div>
 <div class="row">
-    <div class="col-md-12 col-sm-12 col-xs-12">
+    <div class="col-12">
```

Comment:
Just for my knowledge, this doesn't actually change anything, but just removes unnecessary classes, correct?

Reply:
Exactly. Also `col-xs-12` is BS3, `col-12` is BS4.